experiments/browser_based_querying/patches/graphql-language-service@5.1.7.patch

```diff
@@ -1,15 +1,36 @@
 diff --git a/esm/interface/getAutocompleteSuggestions.js b/esm/interface/getAutocompleteSuggestions.js
-index ac159d132d095a789b966627d315f4238c79ab60..a3be2d27c9d5c7d6fd3745093301a07d28e79a8d 100644
+index ac159d132d095a789b966627d315f4238c79ab60..fa7f614cb2feb8a90bca66bf2c6dcb6763c4a1f3 100644
 --- a/esm/interface/getAutocompleteSuggestions.js
 +++ b/esm/interface/getAutocompleteSuggestions.js
-@@ -234,7 +234,12 @@ export function getAutocompleteSuggestions(schema, queryText, cursor, contextTok
+@@ -63,6 +63,9 @@ const hasTypeSystemDefinitions = (sdl) => {
+     }
+     return hasTypeSystemDef;
+ };
++const primitives = new Set(['Int', 'Float', 'String', 'Boolean'])
++const edgeDirectives = new Set(['fold', 'recurse', 'optional', 'transform'])
++const propertyDirectives = new Set(['filter', 'tag', 'output', 'transform'])
```

Comment:
It won’t prevent it, but we will not autocomplete properties after transform yet, that seems like it will be much more involved to add, although it can be explored.

Reply:
if a user _started_ typing `@filter` after a `@transform` on an edge, will it continue the completion from there?

Imagine they are here:
```graphql
{ Root { edge @fold @transform(op: "count") @fil
```

Will it autocomplete `@filter` and its arguments?

If yes, that's great and let's merge this. If not, we should maybe add all the possible directives that could appear on edges here instead — it's not great if the autocomplete suggests broken code, but it might be worse if it pretends to not understand completely valid code...

Reply:
No it will not, I removed the edge filtering code for now and just left a comment about this case.